paddle/operators/chunk_eval_op.cc

```diff
@@ -70,6 +80,14 @@ class ChunkEvalOpMaker : public framework::OpProtoAndCheckerMaker {
               "sensitivity) of chunks on the given mini-batch.");
     AddOutput("F1-Score",
               "(float). The evaluated F1-Score on the given mini-batch.");
+    AddOutput(
+        "NumInferChunks",
```

Comment:
can NumInferChunks be different with NumLabelChunks?

Reply:
Yes, `NumInferChunks` can be different with`NumLabelChunks` since they indicate the number of chunks in inference data and labeled data respectively.